build/periodic.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/sh
+
+# Validate the pipeline is up to date and that no failed prow jobs exist
+
+COMPONENT_ORG=stolostron
+CHECK_RELEASES="2.3 2.4 2.5"
```

Comment:
For a future enhancement, we could use https://github.com/stolostron/release/blob/master/CURRENT_RELEASE to get the current release and create these versions dynamically.